### PR TITLE
#1345 Fixed insensitive or inconsiderate language

### PR DESCRIPTION
**Pull request by hatwell**
Ran linter and improved insensitive or inconsiderate language. Removed some unnecessary 'just's and 'simply's and tried to use more appropriate or descriptive words where possible.

### CODE REVIEW

File: website/versioned_docs/version-0.58/communication-android.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-id: version-0.58-communication-android
-title: Communication between native and React Native
-original_id: communication-android
+
+id: version-0.58-communication-android title: Communication between native and React Native original_id: communication-android
```
Comment (rickhanlonii) [inline website/versioned_docs/version-0.58/communication-android.md]: @hatwell this change is breaking the deploy

File: docs/typescript.md [hunk 2]
```diff
@@ -89,68 +89,70 @@ import React from 'react';
 import {Button, StyleSheet, Text, View} from 'react-native';
 
 export interface Props {
-    name: string;
-    enthusiasmLevel?: number;
```
Comment (rickhanlonii) [inline docs/typescript.md]: I'm also curious - how did you update the unrelated whitespace changes?

Did you do these manually or with a script?

Comment (hatwell) [reply docs/typescript.md]: Hello, @rickhanlonii, I think this is something my editor has 'helpfully' done for me - I'll take a look and get the tests green then resubmit.

Comment (rickhanlonii) [reply docs/typescript.md]: Ah yeah it may be a default prettier config that runs on markdown files

**Comment by hatwell** [comment]
I have fixed that breaking change now :)

File: website/versioned_docs/version-0.58/panresponder.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ title: PanResponder
 original_id: panresponder
 ---
 
-`PanResponder` reconciles several touches into a single gesture. It makes single-touch gestures resilient to extra touches, and can be used to recognize simple multi-touch gestures.
+`PanResponder` reconciles several touches into a single gesture. It makes single-touch gestures resilient to extra touches, and can be used to recognize striaghtforward multi-touch gestures.
```
Comment (rickhanlonii) [inline website/versioned_docs/version-0.58/panresponder.md]:
```suggestion
`PanResponder` reconciles several touches into a single gesture. It makes single-touch gestures resilient to extra touches, and can be used to recognize straightforward multi-touch gestures.
```

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (rickhanlonii) [inline yarn.lock]: Can you remove this file? I think you may have accidentally ran `yarn` in the root (I do this all the time in this repo)

Comment (hatwell) [reply yarn.lock]: yep, just committed (thank you )

**Comment by rickhanlonii** [comment]
Oh no, I'm sorry I think you removed the wrong yarn.lock 🙈

**Comment by hatwell** [comment]
```Build-agent version 1.0.17376-9104d073 (2019-10-02T15:59:42+0000)
Docker Engine Version: 18.09.6
Kernel Version: Linux 0aa454aaed4d 4.15.0-1043-aws #45-Ubuntu SMP Mon Jun 24 14:07:03 UTC 2019 x86_64 Linux
Starting container circleci/node:8
  image cache not found on this host, downloading circleci/node:8

  Error pulling image circleci/node:8: Error response from daemon: received unexpected HTTP status: 503 Service Unavailable... retrying
  image cache not found on this host, downloading circleci/node:8

  Error pulling image circleci/node:8: Error response from daemon: received unexpected HTTP status: 503 Service Unavailable... retrying
  image cache not found on this host, downloading circleci/node:8

  Error pulling image circleci/node:8: Error response from daemon: received unexpected HTTP status: 503 Service Unavailable... retrying
  image cache not found on this host, downloading circleci/node:8

  Error pulling image circleci/node:8: Error response from daemon: received unexpected HTTP status: 503 Service Unavailable... retrying
  image cache not found on this host, downloading circleci/node:8

  Error pulling image circleci/node:8: Error response from daemon: received unexpected HTTP status: 503 Service Unavailable... retrying
  image cache not found on this host, downloading circleci/node:8

Error response from daemon: received unexpected HTTP status: 503 Service Unavailable```

This is the error on the latest CI run, it's some problem with building the container? I'm not sure what to do to resolve it

**Comment by rickhanlonii** [comment]
@hatwell if you could give me access to edit this PR I can wrap this up for you!

**Comment by hatwell** [comment]
@rickhanlonii I am so sorry but I don't know how to grant someone else access. If it isn't totally obvious, this is my first time doing this :)

**Comment by hatwell** [comment]
@rickhanlonii i think i figured it out!
